ang/civicase/case/directives/case-overview.directive.js

```diff
@@ -47,7 +47,10 @@
     var caseTypes = CRM.civicase.caseTypes;
     var caseTypeCategories = CRM.civicase.caseTypeCategories;
 
-    $scope.caseStatuses = CRM.civicase.caseStatuses;
+    $scope.caseStatuses = _.chain(CRM.civicase.caseStatuses)
```

Comment:
Please move this to the next line
```
$scope.summaryData = [];
$scope.caseStatuses  = ...
```

idea is to have the single line variable first.